projects/objects/computers/protos/ComputerMouse.proto

```diff
@@ -13,63 +13,63 @@ PROTO ComputerMouse [
+DEF TOPMOUSE Shape {
```

Comment:
There is a bug in the script, the indentation doesn't work properly for certain nodes

Reply:
Thanks, didn't notice. I checked the others as well and it appears to be the only case.